packages/js/src/migrations/update-16-6-0/explicitly-set-projects-to-update-buildable-deps.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  formatFiles,
+  getProjects,
+  Tree,
+  updateProjectConfiguration,
+} from '@nx/devkit';
+
+const executors = new Set([
+  '@nx/js:swc',
+  '@nrwl/js:swc',
+  '@nx/js:tsc',
+  '@nrwl/js:tsc',
+]);
+
+export default async function (tree: Tree) {
+  const projects = getProjects(tree);
+
+  for (const [projectName, project] of projects) {
```

Comment:
`forEachExecutorOptions` might be easier for this case like https://github.com/nrwl/nx/blob/master/packages/node/src/migrations/update-16-3-1/update-webpack-executor.ts

Reply:
It was a conscious decision not to use it to avoid another loop for the executors. The `forEachExecutorOptions` function works for a single executor, so I would need to loop the executors to use that function.